app/controllers/application_controller.rb

```diff
@@ -1,2 +1,25 @@
+# frozen_string_literal: true
+
+# ApplicationController
 class ApplicationController < ActionController::Base
+  helper_method :current_user
+
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    session[:user_id].nil? ? nil : User.find(session[:user_id])
+  end
+
+  def logged_in?
```

Comment:
logged in and current_user are having the almost same input and as such you should keep only one of those methods.